plugin/src/runtime-support.cc

```diff
@@ -259,12 +259,19 @@ std::string rt::fuid() {
     throw ValueUnavailable("fuid() not available in current context");
 }
 
-void rt::file_begin(const std::optional<std::string>& mime_type) {
+std::string rt::file_begin(const std::optional<std::string>& mime_type) {
     auto cookie = static_cast<Cookie*>(hilti::rt::context::cookie());
     auto* fstate = _file_state(cookie);
     ++fstate->file_id;
     fstate->mime_type = mime_type;
 
+    // Feed an empty chunk into the analysis to force creating the file state inside Zeek.
+    _data_in("", 0);
```

Comment:
With this we should now be able to remove the block at l.279.

Reply:
oh, yeah, that's a rebase artifact, didn't get cleaned up.